### PR TITLE
Release notes for v4.5.

### PR DESCRIPTION
**Pull request by EFM-Bobby**
Release notes for EFM 4.5.

Jira ticket https://enterprisedb.atlassian.net/browse/EFM-1526

We should release by the 31st at the latest.

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**

- [X] This PR adds new content
- [ ] This PR changes existing content
- [ ] This PR removes existing content
- [ ] This PR is for a release, please add this tag:

### CODE REVIEW

**Comment by EFM-Bobby** [comment]
The version should be live Monday for final testing, and we can officially release and publish this announcement after that.

There is also a PR for some user's guide changes related to this release, which can go in any time with/after the announcement.

File: product_docs/docs/efm/4/efm_rel_notes/05_efm_45_rel_notes.mdx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+| Bug Fix | Failover Manager will now properly choose a standby to promote in rare cases of LSN values being compared incorrectly. [Support ticket #83186] |
+| Bug Fix | Failover Manager now handles extra whitespace in properties file when running the `efm` command for cluster status. |
+| Bug Fix | Failover Manager will no longer fail to promote in rare cases where the primary agent is running slowly. [Support ticket #79847] |
+| Bug Fix | Failover Manager properly follows the promotable=false value of a standby that is restarted after being made promotable manually through the `efm` command. [Support ticket #83192] |
+| Bug Fix | When using physical replication slots, Failover Manager will now drop and recreate slots on standbys when advancing the LSN values on them if they have an `xmin` value set. [Support ticket #83910] |
+| Bug Fix | Systemd status will now show "inactive" status instead of "failed" after being used to stop a Failover Manager agent. |
```
Comment (EFM-Bobby) [inline product_docs/docs/efm/4/efm_rel_notes/05_efm_45_rel_notes.mdx:16]: There is no support ticket for this one. It's something that came up internally.

File: product_docs/docs/efm/4/efm_rel_notes/index.mdx [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
+including minor releases and patches. These release notes
+cover what was new in each release. For new functionality introduced
+in a minor or patch release, there are also indicators in the content
 about the release that introduced the feature.
 
-| Version | Release Date | 
-| ------- | ------------ | 
+| Version | Release Date |
+| ------- | ------------ |
+| [4.5](05_efm_45_rel_notes) | 2022 Aug 31 |
```
Comment (EFM-Bobby) [inline product_docs/docs/efm/4/efm_rel_notes/index.mdx:12]: Ugh, the above are whitespace only changes that I can't get my IDE to not make. Hopefully we don't actually need a blank space at the end of those lines.

File: product_docs/docs/efm/4/efm_rel_notes/index.mdx [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
+The Failover Manager documentation describes the latest version of Failover Manager 4,
+including minor releases and patches. These release notes
+cover what was new in each release. For new functionality introduced
+in a minor or patch release, there are also indicators in the content
 about the release that introduced the feature.
 
-| Version | Release Date | 
-| ------- | ------------ | 
+| Version | Release Date |
+| ------- | ------------ |
```
Comment (EFM-Bobby) [inline product_docs/docs/efm/4/efm_rel_notes/index.mdx:11]: Ugh, the above are whitespace only changes that I can't get my IDE to not make. Hopefully we don't actually need a blank space at the end of those lines.

**Review by kelpoole — APPROVED** [review]
I fixed the title and other references to 4.4 in the 4.5 release notes and fixed a minor typo

**Pull request merged**